_src/pattern-library/sass/components/_layouts.scss

```diff
@@ -0,0 +1,708 @@
+// ------------------------------
+// edX Pattern Library: Components - Layouts
+
+// About: Contains base settings and options for the preset default Pattern Library UI Layouts.
+// ----------------------------
+
+// #CONFIG
+// #GLOBALS
+// #UTILITY
+// #CANNED - IN VIEW
+// #CANNED - VIEW
+
+// ----------------------------
+
+// ----------------------------
+// #CONFIG
+// ----------------------------
+$susy-flush:            layout($susy-default inside); // set susy to make columns flush
```

Comment:
NOTE: setting specific Susy layout maps (http://susy.readthedocs.org/en/latest/settings/#layout-function) for different layout cases.